test/images/resource-consumer/README.md

```diff
@@ -0,0 +1,66 @@
+# Resource Consumer container for testing
+
+Resource Consumer is a tool for testing kubernetes autoscaling.
+It can help with tests for:
+- cluster size autoscaling
+- horizontal autoscaling of pod - changing the size of replication controller
+- vertical autoscaling of pod - changing its resource limits
+
+image: gcr.io/google_containers/resource_consumer:beta
+
+The container consumes specified amount of resources:
```

Comment:
Start with explaining what resource consumer: http server which accept some requests
and then describe in details what the specification of requests is.